### ISSUE TITLE
Timer capture/compare control register should be TACCTLx

### ISSUE THREAD

**Issue by tgtakaoka** [dlbeer/mspdebug#67]
`simio info <timer>` command reports timer capture/compare control register as TACTL, but it should be TACCTL. TACTL is for Timer A control register.

```
$ mspdebug sim
...
(mspdebug) simio add timer TA3
Added new device "TA3" of type "timer".
(mspdebug) simio info TA3
Base address: 0x0160
IV address:   0x012e
IRQ0:         9
IRQ1:         8

TACTL:        0x0000
TAR:          0x0000
TAIV:         0x00

Channel  0, TACTL = 0x0000, TACCR = 0x0000
Channel  1, TACTL = 0x0000, TACCR = 0x0000
Channel  2, TACTL = 0x0000, TACCR = 0x0000
(mspdebug) 
```

**Comment by tgtakaoka** [comment]
This issue should be fixed by #62.

**Comment by tgtakaoka** [comment]
Fixed.
